src/modules/rlm_lua/lua.c

```diff
@@ -466,6 +466,36 @@ static int _lua_list_iterator_init(lua_State *L)
 	return 1;
 }
 
+/*
+ * 	Initialise the table "fr." with all valid return codes.
+ */
+static int _lua_rcode_table_newindex(lua_State *L)
+{
+	lua_pushstring(L, "You can't modify the table 'fr.*' (read-only)");
```

Comment:
Why is this pushing a string, it can just call the logging functions directly and produce an error??